handlers/jobs.go

```diff
@@ -160,7 +160,8 @@ func createK8sJob(currentAction string, inputData string, accessFormat string, a
 	var ttl int32 = 3600 // Keep job pods for 1 hour by default
 	labels := make(map[string]string)
 	labels["app"] = "sowerjob"
-	labels["username"] = username
+	safeUsername := sanitizeLabelValue(username)
```

Comment:
I'm trying to wrap my head around, but it seems these are maybe not the best place to call the sanitization function? There are already some sanitization happening in [here](https://github.com/uc-cdis/sower/blob/master/handlers/sower.go#L225-L237). And if you trace it up, it looks like that function `getEmailFromToken()` isn't really doing what its name suggests. Instead it is more like `getUserIDFromToken`. And the returned value from that function as being feed into other functions, and get sanitized by the `sanitizeLabelValue()` function again
So I think it might be better to sanitize the user ID in that function, and rename the function and its returned varaibles to make it more clear